woocommerce/wc-admin
# Display modal with more info about the new homescreen

Fixes #4590

This implements a paged modal via the Gutenberg `Guide` component. Some styling adjustment was needed to match the designs exactly.

This also introduces 3 SVG based illustrations for display in the modal. According to [this comment](https://github.com/woocommerce/woocommerce-admin/issues/4590#issuecomment-649669897) there is a need to color them based on the theme at some point, but based on a current lack of requirements for that, I felt that could be done at a separate time (that was suggested in the comment as well).

### Accessibility




### Screenshots

![welcome-modal](https://user-images.githubusercontent.com/1281828/88874119-66249680-d272-11ea-8d58-645953660764.gif)



### Detailed test instructions:

- Go to the homescreen on a site that has never dismissed the modal before
- You should see the new welcome modal
- click through the pages of the modal to see all the content and illustrations. Confirm styling is according to [the designs](https://www.figma.com/file/JH9XMFUCOjfXdr3N09AHRD/On-boarding-iterations-June-'20?node-id=545%3A20969).
